content/maxwell1_fundamentals/dipole_sources_in_homogeneous_media/electric_dipole_frequency/index.rst

```diff
@@ -4,50 +4,73 @@ Harmonic Electrical Current Dipole
 
-Let us now examine the source current which results from a harmonic electric dipole source.
-For a source current of magnitude :math:`I` which lies in the :math:`\hat x` direction along a wire segment of length :math:`\Delta s`, the source current density is given by:
 
-.. math::
-	\mathbf{J (r)} = \hat x I \Delta s \Bigg [ \frac{\textrm{u}\big (x + \frac{\Delta s}{2} \big ) - \textrm{u} \big ( x - \frac{\Delta s}{2} \big )}{\Delta s} \Bigg ] \delta (y) \delta (z)
+
+The harmonic electrical current dipole can be thought of as an infinitesimally short length of wire which carries a harmonic current.
+The strength of the source is therefore defined by a harmonic dipole moment :math:`\mathbf{p}(\omega)`.
+For a harmonic current dipole defined by length :math:`ds` and harmonic current :math:`\mathbf{I} (\omega) = \mathbf{I}e^{i\omega t}`, the dipole moment is given by:
 
 
-where :math:`u(x)` is the unit step function and :math:`\delta (x)` is the Dirac delta function.
-By letting :math:`\Delta s \rightarrow ds`, the source current density for a harmonic electric dipole in the :math:`\hat x` direction is given by:
 
 .. math::
-	\mathbf{J(r)} = \hat x I ds \delta (x) \delta (y) \delta (z)
+	\mathbf{p}(\omega) = \mathbf{p} \, e^{i\omega t} = \mathbf{I} ds \, e^{i\omega t}
+	:name: p_harmonic_def
+
 
+where :math:`\mathbf{p} = \mathbf{I}ds` is the vector amplitude of the dipole moment.
+When formulating Maxwell's equations in the frequency domain, :math:`e^{i\omega t}` is generally suppressed.
+As a result, the source term for the harmonic electrical current dipole is given by:
 
-By analogy of the previous equation, the source current density for a harmonic electric dipole in any direction can be expressed as:
 
 .. math::
-	\mathbf{J(r)} = \mathbf{I}ds \, \delta (x) \delta (y) \delta (z) = \mathbf{p} \, \delta (x) \delta (y) \delta (z)
+	\mathbf{J_e^s} = \mathbf{I}ds \, \delta (x) \delta (y) \delta (z)
+	:name: Je_harmonic_def
 
 
+where :math:`\delta (x)` is the Dirac delta function.
+By including the source term, Maxwell's equations in the frequency domain are given by:
+
+
+.. math::
+	\begin{split}
+	&\nabla \times \mathbf{E_e} + i \omega \mu \mathbf{H_e} = 0  \\
+	\nabla \times \mathbf{H_e} - & (\sigma  + i\omega \varepsilon ) \mathbf{E_e} = \mathbf{I}ds \, \delta(x) \delta(y) \delta(z)
+	\end{split}
+	:name: p_Maxwells_harmonic
 
-**Needs Image** with some description.
 
 
+where subscripts :math:`_e` remind us that we are considering an electric source.
+The source current is responsible for generating a primary current density (and thus an electric field) in the surrounding region (:numref:`ElecDipole`).
+However, the :ref:`Ampere-Maxwell equation<ampere_maxwell_differential_frequency>` states that harmonic electric fields as well as free currents generate magnetic fields.
+In addition, the harmonic nature of the magnetic fields should produce secondary electric fields according to :ref:`Faraday's law<faraday_differential_frequency>`.
```

Comment:
This link is not rendering properly. Perhaps you need a space after law? 
![image](https://cloud.githubusercontent.com/assets/6361812/23598132/1f0eadb6-01ed-11e7-910d-f12add8f463e.png)